core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/scan/text/TextScanSourceOpDesc.scala

```diff
@@ -0,0 +1,97 @@
+package edu.uci.ics.texera.workflow.operators.source.scan.text
+
+import com.fasterxml.jackson.annotation.{
+  JsonIgnoreProperties,
+  JsonProperty,
+  JsonPropertyDescription
+}
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize
+import com.kjetland.jackson.jsonSchema.annotations.{
+  JsonSchemaInject,
+  JsonSchemaString,
+  JsonSchemaTitle
+}
+import edu.uci.ics.amber.engine.architecture.deploysemantics.layer.OpExecConfig
+import edu.uci.ics.texera.workflow.common.metadata.annotations.HideAnnotation
+import edu.uci.ics.texera.workflow.common.tuple.schema.{
+  Attribute,
+  AttributeType,
+  OperatorSchemaInfo,
+  Schema
+}
+import edu.uci.ics.texera.workflow.operators.source.scan.ScanSourceOpDesc
+
+import java.io.{BufferedReader, FileReader}
+import scala.collection.convert.ImplicitConversions.`iterator asScala`
+
+@JsonIgnoreProperties(value = Array("limit", "offset"))
```

Comment:
move this to individual property, it's harder to read on class.

Reply:
also, why are we ignoring limit and offset?

Reply:
oh then nvm. it's the issue of inheritance. you can leave a comment and explain this, for future refactor to consider.